crates/network-scanner-net/Cargo.toml

```diff
@@ -8,9 +8,10 @@ publish = false
 [dependencies]
 anyhow = "1.0.79"
 crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
 futures = "0.3.30"
 parking_lot = "0.12.1"
-polling = { git = "https://github.com/smol-rs/polling", rev = "62430fd56e668559d08ca7071ab13a0e116ba515" }
+polling = { git = "https://github.com/irvingoujAtDevolution/polling.git", branch = "fix-linux-tcp-fail" }
```

Comment:
Wasn't aware the problem of Linux side untile the CI reported me error. :(. 

This is currently necessary for linux TCP to work, see line 237 for reason

Reply:
Can you link to the issue / pull request you created on their side?

Reply:
the problem on polling is that they are trying to have a platform indepent interface for all polling mechanisms, so I guess they would particularly against have something like this. I have a better idea in mind to how to implement this so it is more acceptable for both us and them, I'll put on a update shortly.

Reply:
https://github.com/smol-rs/polling/actions/runs/7819053683/job/21330660851?pr=189